enumerable.rb

```diff
@@ -0,0 +1,76 @@
+module Enumerable
+  def my_each
+    length.times do |x|
+      yield(self[x])
+    end
+  end
+
+  def my_each_with_index
```

Comment:
- [ ] **enumerables #my_each_with_index should return an enumerator if no block is given**
Your Output: 
```
LocalJumpError:
       no block given (yield)
```

- [ ] **enumerables #my_each_with_index when a block is given when self is an array returns the array itself after calling the given block once for each element in self**
```
expected: [1,2,3,4,5]
            got: 15
```

- [ ] ** enumerables #my_each_with_index when a block is given when self is a range returns the range itself after calling the given block once for each element in self**
```
NameError:
       undefined local variable or method `length' for 5..50:Range
```

- [ ] **enumerables #my_each_with_index when a block is given when self is a range behaves like #each_with_index calls the given block once for each element in self**
```
 NameError:
       undefined local variable or method `length' for 5..50:Range
```
- [ ] ** enumerables #my_each_with_index when a block is given when self is a hash behaves like #each_with_index calls the given block once for each element in self **
``` 
expected: "Num: [:a, 1], idx: 0\nNum: [:b, 2], idx: 1\nNum: [:c, 3], idx: 2\nNum: [:d, 4], idx: 3\nNum: [:e, 5], idx: 4\n"
             got: "Num: , idx: 0\nNum: , idx: 1\nNum: , idx: 2\nNum: , idx: 3\nNum: , idx: 4\n"
```

Reply:
Hello, I believe that the changes required are done.